pkg/platform/aws/aws.go

```diff
@@ -287,6 +288,28 @@ func (c *config) checkValidConfig() hcl.Diagnostics {
 	return diagnostics
 }
 
+// checkDifferentPortForMuliplePools checks if user is using multiple worker
+// pools, then they should have lb_http_port and lb_https_port flag set
+// with different values.
+func (c *config) checkDifferentPortForMuliplePools() hcl.Diagnostics {
+	var diagnostics hcl.Diagnostics
+
+	// check only if using multiple worker pools.
+	if len(c.WorkerPools) > 1 {
+		for _, wp := range c.WorkerPools {
+			if wp.LBHTTPPort == 0 && wp.LBHTTPSPort == 0 {
```

Comment:
I guess we can collect the ports in map to ensure they are unique?

Reply:
Yeah,  just thought of that after seeing failing build